lib/shared/snackbar.dart

```diff
@@ -15,55 +15,65 @@ void showSnackbar(
   IconData? trailingIcon,
   void Function()? trailingAction,
 }) {
-  int wordCount = RegExp(r'[\w-]+').allMatches(text).length;
-  SnackBar snackBar = SnackBar(
+  final ThemeData theme = Theme.of(context);
+  final int wordCount = RegExp(r'[\w-]+').allMatches(text).length;
+
+  GetSnackBar snackBar = GetSnackBar(

```

Comment:
Why not use the default `snackbar `?

Reply:
Please see the PR description for my rationale behind this change! Especially watch the before/after videos, and see how the code is simplified without having to worry about context and scaffolds! 😊